src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const dayRate = 40;
+  const lowerDiscount = 20;
+  const basicDiscount = 50;
+
+  if (days >= 7) {
+    total = days * dayRate - basicDiscount;
+  } else if (days >= 3) {
+    total = days * dayRate - lowerDiscount;
+  } else {
+    total = days * dayRate;
+  }
```

Comment:
Don't repeat yourself: you repeated this action `days * dayRate` 3 times, let's complete it before if / else statements